Doc/library/bz2.rst

```diff
@@ -112,6 +110,7 @@ All of the classes in this module may safely be accessed from multiple threads.
 
    .. deprecated:: 3.0
```

Comment:
Should not the whole directive be removed?

Reply:
done